docs/Config.md

```diff
@@ -435,6 +435,8 @@ gui:
 
 For all possible keybinding options, check [Custom_Keybindings.md](https://github.com/jesseduffield/lazygit/blob/master/docs/keybindings/Custom_Keybindings.md)
 
+You can disable certain key bindings by specifying `null`.
```

Comment:
added.

>also should this be the null keyword or the string 'null'?

I don't think it is necessary to accept the string `'null'` since `null` keyword is clear enough in intent.